client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -150,7 +152,7 @@ public abstract CelebornInputStream readPartition(
   public abstract CelebornInputStream readPartition(
       int shuffleId, int partitionId, int attemptNumber) throws IOException;
 
-  public abstract boolean unregisterShuffle(int shuffleId, boolean isDriver);
+  public abstract boolean unregisterShuffle(int shuffleId);
```

Comment:
Little confuse for the method(unregisterShuffle) now, better to change it.

Reply:
Maybe cleanupShuffle

Reply:
Done